ui/app/src/modules/Content/Dependencies/DependenciesDialog.tsx

```diff
@@ -15,7 +15,7 @@
  */
 
 import React, { PropsWithChildren, useCallback, useEffect, useMemo, useState } from 'react';
-import { SandboxItem } from '../../../models/Item';
+import { DetailedItem, SandboxItem } from '../../../models/Item';
```

Comment:
- Please check if dialog can work with a BaseItem so that it could receive Sandbox or Detailed item alike.
- Looks like you've completely removed SandboxItem so make sure to clean imports

Reply:
we agreed to use detailed Item now.

Reply:
The dialogs should use the Detailed item state from redux, there are pending work for this.